test/verify/check-shell-multi-machine-key

```diff
@@ -139,7 +139,7 @@ class TestMultiMachineKeyAuth(testlib.MachineCase):
         try:
             m1.execute("ps xa | grep ssh-agent | grep -v grep")
         except subprocess.CalledProcessError:
-            assert False, "No running ssh-agent found"
+            self.fail("No running ssh-agent found")
```

Comment:
This is a good fix, just the commit message "add specific noqa comments for PT015" is now wrong. But no need for another round for this.